### PR TITLE
adds an error messsage to force removing something from the grid

### PR DESCRIPTION
**Pull request by Kylerace**
## About The Pull Request
saw another error message with something being stuck in the spatial grid in #73760 and looked at the code again. not sure what the issue is but with this it should be doable now
## Why It's Good For The Game
hopefully leads to a fix

### CODE REVIEW

File: code/controllers/subsystem/spatial_gridmap.dm [hunk 1]
```diff
@@ -51,6 +51,52 @@
...
+			channels = "[channels], atmos"
+		number_of_channels += 1
+
+	if(number_of_channels == 0)
+		channels = "channels: NONE"
+
+	return "CELL{[coords], [channels]}"
+
+#ifdef UNIT_TESTS
+/mob/living/carbon/human/lizard/trolls_the_maintainer
```
Comment (Kylerace) [inline code/controllers/subsystem/spatial_gridmap.dm:86]: ignore this, just testing that it throws the error in create_and_destroy

**Comment by Kylerace** [comment]
The error has errors

**Comment by Kylerace** [comment]
the error has no errors (this is an error)

File: code/controllers/subsystem/spatial_gridmap.dm (hunk 1 above)
Comment (Kylerace) [reply code/controllers/subsystem/spatial_gridmap.dm:87]: :trollface:

**Comment by Mothblocks** [comment]
@Kylerace updates?

**Comment by Kylerace** [comment]
Working on college applications I'll finish it after the 15th

**Comment by ZephyrTFA** [comment]
ayo, good luck bud

**Pull request closed without merging**